### DISCUSSION TITLE
change kernel in Volterra_IDE

### DISCUSSION THREAD

**Discussion by Bensayah** [lululxvi/deepxde#822] [Q&A]
I need to use the kernel s**(-0.2)*(t-s)**(-0.2) in Volterra_IDE  it gives nan in Train loss.
Is it possible to handle this kernel?

**Comment by lululxvi** [comment]
The issue could be that when s = 0 or s = t, then the kernel blows up.

**Reply by Bensayah** [reply]
but the definite integral is finite

**Reply by lululxvi** [reply]
the computation of 0**(-0.2) returns nan
